src/app/backend/resource/deployment/deploy.go

```diff
@@ -106,6 +106,18 @@ type AppDeploymentFromFileSpec struct {
 	Validate bool `json:"validate"`
 }
 
+// AppDeployRollBackSpec is a specification for deployment rollback from an specific revision number
+type AppDeployRollBackSpec struct {
```

Comment:
s/AppDeployRollBackSpec/DeploymentRollbackSpec

Reply:
Move this structure to `rollback.go` after renaming.

Reply:
okay, I will modified it later.

Reply:
OK, I'll modify it later，thank you both~